frontend/src/components/common/header/styles.module.scss

```diff
@@ -1,6 +1,9 @@
 .headerWrapper {
+  position: fixed;
```

Comment:
use sticky instead

Reply:
Should I use sticky only for header? Or for authorized menu as well?